crowbar_framework/lib/crowbar_pacemaker_proxy.rb

```diff
@@ -54,6 +63,24 @@ def is_cluster?(element)
     result
   end
 
+  def is_remotes?(element)
```

Comment:
Style/PredicateName: Rename `is_remotes?` to `remotes?`. (https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)